amd/src/quizgame.js

```diff
@@ -396,33 +463,49 @@ define(['jquery','core/yui', 'core/notification', 'core/ajax'], function($, Y, n
         }
     }
 
-    function Rectangle(left, top, width, height)
-    {
+    /**
+     * Helper Function ??
```

Comment:
```suggestion
     * Old style JS constructor for storing information about a rectangle shape
```

This does need to be replaced with a proper class definition, assuming moodle supports es6 these days